src/main/java/bio/terra/service/snapshotbuilder/utils/ConceptChildrenQueryBuilder.java

```diff
@@ -22,26 +24,49 @@ public static String buildConceptChildrenQuery(
     FieldVariable nameFieldVariable = conceptTableVariable.makeFieldVariable("concept_name");
     FieldVariable idFieldVariable = conceptTableVariable.makeFieldVariable("concept_id");
 
-    TablePointer ancestorTablePointer =
-        TablePointer.fromTableName("concept_ancestor", tableNameGenerator);
-    TableVariable ancestorTableVariable = TableVariable.forPrimary(ancestorTablePointer);
+    TablePointer relationshipTablePointer =
+        TablePointer.fromTableName("concept_relationship", tableNameGenerator);
+    TableVariable relationshipTableVariable = TableVariable.forPrimary(relationshipTablePointer);
     FieldVariable descendantFieldVariable =
```

Comment:
For better readability, what do you think about removing the type names from the variables? The code is really about tables and fields which IMO are more naturally named using the table and column names. To me, having the types in the names adds clutter but doesn't make the code clearer.
```suggestion
    TableVariable relationship = TableVariable.forPrimary(relationshipTablePointer);
    FieldVariable descendent =
```

Or maybe `conceptRelationship` for the table name.